test/ex_cell/controller_test.exs

```diff
@@ -6,14 +6,22 @@ defmodule ExCell.ControllerTest do
   end
 
   test "cell/2 with controller" do
-    assert ExCell.Controller.cell(%{}, :mock_cell) === [%{}, :mock_cell, "template.html", []]
+    assert ExCell.Controller.cell(%{private: %{}}, :mock_cell) === [
+             %{private: %{phoenix_view: :mock_cell}},
```

Comment:
What does the `private` part do exactly?

Reply:
it's what put_view does, it calls put_private_view on Plug.Conn and sets the chosen view as a value under the key phoenix_view in the private map on the conn.

Reply:
https://github.com/phoenixframework/phoenix/blob/v1.3.3/lib/phoenix/controller.ex#L435